imap/http_caldav.c

```diff
@@ -3730,11 +3754,9 @@ static int caldav_put(struct transaction_t *txn, void *obj,
         goto done;
     }
 
+    construct_hashu64_table(&overrides, 10, 0);
```

Comment:
This magic number seems unnecessarily small. If we can't bother to count the number of overrides in a linear scan of the icalcomponent, then just use some "safer" higher limit like 128 or even 512 here. We've seen Cyrus choke on the rare large size iCalendar choke before.

Reply:
Done.  I split the diff and went with 256.